Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -2908,6 +2908,24 @@ RefPtr<CSSValue> ComputedStyleExtractor::valueForPropertyInStyle(const RenderSty
             return cssValuePool.createIdentifierValue(CSSValueAuto);
         return currentColorOrValidColor(style, style.accentColor());
     }
+    case CSSPropertyAlignTracks: {
+        auto list = CSSValueList::createCommaSeparated();
+        auto alignTracks = style.alignTracks();
+
+        for (auto x : alignTracks)
+            list->append(valueForContentPositionAndDistributionWithOverflowAlignment(x));
+
+        return list;
+    }
+    case CSSPropertyJustifyTracks: {
```

Comment:
I'd combine this with the previous case and use a ternary to pick between alignTracks and justifyTracks